src/v/raft/configuration_manager.cc

```diff
@@ -433,6 +433,39 @@ model::revision_id configuration_manager::get_latest_revision() const {
     return _configurations.rbegin()->second.cfg.revision_id();
 }
 
+int64_t configuration_manager::offset_delta(model::offset o) const {
+    auto it = lower_bound(o);
+
+    if (it == begin()) {
+        /**
+         * iterator points to the first configuration with offset greater
+         * than then requsted one. Knowing an index of that configuration we
+         * know that there was exactly (index -1) configurations with offset
+         * lower than the current one. We can simply subtract one from
+         * index.
+         */
+        return std::max<int64_t>(0, it->second.idx() - 1);
```

Comment:
subtracting 1 from the index, does not explaian the std::max piece, which is the salient point in this PR

Reply:
this was just moved in this PR, it was implemented before in:
- https://github.com/vectorizedio/redpanda/pull/2325